src/api/app/views/staging/staging_projects/_missing_reviews.xml.builder

```diff
@@ -1,5 +1,5 @@
 builder.missing_reviews(count: count) do |missing_review|
   missing_reviews.each do |bs_request|
-    missing_review.entry(id: bs_request[:id], creator: bs_request[:by], state: bs_request[:state], package: bs_request[:package])
+    missing_review.review(id: bs_request[:id], creator: bs_request[:by], state: bs_request[:state], package: bs_request[:package])
```

Comment:
Ouch, it's worse. This is pure copy & paste code

```
  <missing_reviews count="2">
    <entry id="3525778" creator="zaz" state="new" package="zaz"/>
    <entry id="3565150" creator="repo-checker" state="new" package="zaz"/>
  </missing_reviews>
```

Both refer to the same request done by me. So creator is just fake and id is what? the review id?

Reply:
https://build.opensuse.org/request/show/734043

Reply:
Filed as https://github.com/openSUSE/open-build-service/issues/8575 so it's not dropped

Reply:
In my case it looks like:


```
  <missing_reviews count="1">
    <review id="3" creator="obs-factory" state="new" package="hello"/>
  </missing_reviews>
```

the review id is the id of the review:

```
irb(main):003:0> Review.find(3)
  Review Load (1.0ms)  SELECT  `reviews`.* FROM `reviews` WHERE `reviews`.`id` = 3 LIMIT 1
=> #<Review id: 3, bs_request_id: 1, creator: "Admin", reviewer: "Admin", reason: "review it", state: "new", by_user: nil, by_group: "obs-factory", by_project: nil, by_package: nil, created_at: "2019-10-14 11:02:56", updated_at: "2019-10-14 11:02:56", review_id: nil, user_id: nil, group_id: 1, project_id: nil, package_id: nil>
```

Reply:
There is no way API to alter a `Review`, so what do we display the ID for? You can only alter (changereviewstate) the `BsRequest`

Reply:
Then let's fix #8575 right with this PR as well. And don't annotate`by`as creator

Reply:
@coolo to fix #8575 is just a matter of change `id` by `number` ?

Reply:
aehm, IMO the whole line is faulty. If you don't want to go with my preferenced format, the next best format would be `<review request="1" by="group:obs-factory" package="hello"/>` (the review state is pointless - if it wasnt' `new` it wouldn't be missing)

Reply:
ok, but can we handle it in a new PR? Because this one is just about the `entry` and to change the XML layout some further discussion will be necessary @coolo

Reply:
We can handle this somewhere else sure.